src/components/_modal/pages/pickup/PickupSelection.js

```diff
@@ -15,17 +15,16 @@ import { handleOnSelect } from "@/components/_modal/utils";
  * Special component responsible for loading order policy
  * Will not render anything, but needs to be mounted
  */
-function PolicyLoader({ branch, onLoad, pid, requireDigitalAccess }) {
+function PolicyLoader({ branch, onLoad, pid, pids, requireDigitalAccess }) {
   const pickupAllowed = branch?.pickupAllowed;
   let { data } = useData(
     pid &&
```

Comment:
enten burde man tjekke her for pid og pids 
eller endnu bedre: man kunne give policyLoader kun pids med. så behøver man kun at tjekke for pids og man kunne give dem direkte med til branchOrderPOlicy uden ||

Reply:
det er jeg iike helt tryg ved - jeg har lavet sådan en her (pid || pids) && ...